### ISSUE TITLE
Test fields

### ISSUE THREAD

**Issue by bennahugo** [bennahugo/catdagger#1]
- TriA example included on readme
- NGC4993 works pretty well
![image](https://user-images.githubusercontent.com/7934586/53373499-55c20b00-395e-11e9-87b4-258511c45753.png)
- A1300 is pretty bad in terms of noise characteristics in I but it seems to at least tag the worst offenders after a meerkathi 2GC
![image](https://user-images.githubusercontent.com/7934586/53374856-4cd33880-3962-11e9-94ee-95276413c2b2.png)

@KshitijT @ianheywood can you think of more cases or ideas on how to improve this?

**Comment by bennahugo** [comment]
So ofc circinus is a problem child... @kshitijT could you make a stokes V map and see if it does any better? probably don't want to tag this complicated extended emission for a dE
![image](https://user-images.githubusercontent.com/7934586/53376638-b0ac3000-3967-11e9-944b-8deb023e3832.png)

**Comment by KshitijT** [comment]
@bennahugo , yeah, I sort of expected this for Circinus. I guess we need to put in that radius conditional in; to avoid tagging central sources for dE; should help at least in the cases when the complicated sources are at the phase centre.

**Comment by KshitijT** [comment]
Also, I will make a V map of Circinus. :)

**Comment by bennahugo** [comment]
Yea problem is this is not really at phase centre and I've seen dE cases well within FWHM - A1300 above is a case in point. I suppose a circinus case is so bright the residual ~0.3% uncorrected leakage into V will clearly show up as well.

**Comment by KshitijT** [comment]
Hmmm. Circinus IS at the phase centre, in the image I gave you. It would be a problem in other pointings, I agree.

**Comment by IanHeywood** [comment]
This looks very cool. Can you simply sum the pixels within the boundaries as a simple test for DDEs vs incomplete deconvolution? I'd expect the latter to have a significantly larger sum.

**Comment by bennahugo** [comment]
Hmm that is actually a pretty good idea. I'll give it a try

**Comment by IanHeywood** [comment]
Could also try histogram tests (significantly positive mean, skew).

Or do a flood fill above say 3-sigma and look for large contiguous regions.

**Comment by bennahugo** [comment]
Yes I've tried classifying based on kurtosis, but it is extremely sensitive to real flux. I've also tried entropy methods as a first pass, but it makes the sigma cutoff very sensitive to the size of the tiles. I'm yet to try further tests on top of the rms detected regions. Thanks for the suggestions @IanHeywood

I'm also thinking I should add @o-smirnov's trick of masking out model images based on pybdsf fitted gaussians tagged with dEs. pybdsm fails hard to fit physical spectra - even when you give it intrinsic maps. @KshitijT and I've been scratching our heads on how to improve the spectra from artificially steepened spectra. This is especially prevalent for compact faint sources and causes holes in the subtracted residual maps. I'm thinking it is best to leave pybdsm island fitting to very high tolerances and model the rest of the faint and extended structure with clean components and multiscale clean to ensure model completion on these very complicated fields.

**Comment by KshitijT** [comment]
@bennahugo , yup, we just need to remove the clean-component modelled sources out of the lsm and vice versa. Not that I am very happy with pybdsm spectra for strong sources either.

**Comment by bennahugo** [comment]
Yup then again you want to have a DFT on bright compact emission beyond the FWHM... so it is probably a good idea to leave them in lsm format

**Comment by bennahugo** [comment]
Ok I've implemented positivity and skewness filters on the regions, as well as exclusion zones. This works quite well. It neatly avoids circinus for example and targets the point sources if the regions are not very large with respect to the targets (ie. sigma too low)

**Comment by bennahugo** [comment]
Implemented masked dE subtraction from FITS and dIE removal from lsm, so this should be all we need to get a semi automatic dIE pipeline going! :)

**Comment by bennahugo** [comment]
Version 0.2.0 is on PyPI
